CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### 💫 New checkers
 
 - `unmentioned-parameter`: Provide error message when a function parameter is not mentioned by name in the function's docstring. By default, this checker is disabled.
+- `redundant-condition`: Provide error message when a conditional statement within a function is guaranteed true. This checker requires `z3` option to be turned on.
```

Comment:
These should be moved into the `unreleased` section (note that I made a version update recently, so this part of the changelog now refers to 2.9.0).